python/ray/tests/test_advanced_5.py

```diff
@@ -194,16 +193,17 @@ def test_worker_lease_reply_with_resources(ray_start_cluster_enabled):
     cluster = ray_start_cluster_enabled
     cluster.add_node(
         memory=2000 * 1024 ** 2,
+        num_cpus=1,
         _system_config={
             "gcs_resource_report_poll_period_ms": 1000000,
             "gcs_actor_scheduling_enabled": True,
         },
     )
-    node2 = cluster.add_node(memory=1000 * 1024 ** 2)
+    node2 = cluster.add_node(memory=1000 * 1024 ** 2, num_cpus=1)
     ray.init(address=cluster.address)
     cluster.wait_for_nodes()
 
-    @ray.remote(memory=1500 * 1024 ** 2)
+    @ray.remote(memory=1500 * 1024 ** 2, num_cpus=0.01)
```

Comment:
The update here is a little bit wired. Why do we need to change the test case like this to make it work?

Reply:
For this test only, we may not need this change. I just did not want the tested normal task and actor depend on the default cpu requirement (which is 1 or 0). Besides, if we don't specify `num_cpus`, the task's RequiredResources and RequiredPlacementResources would be different, which is not preferred.